solutions/on-demand-isr/pages/index.tsx

```diff
@@ -28,7 +28,7 @@ const ProductCard: React.VFC<{ product: Product }> = ({ product }) => {
     >
       <Image
         layout="responsive"
-        width="100%"
```

Comment:
Is this change ok?

Reply:
The build was failing because of this change, TS was complaining so I changed it to this.

This the last deployment in case you want to take a look: https://solutions-on-demand-dxkfxt05k.vercel.sh/